docs/docs/60-new-docs/40-operator-guide/40-security/40-managing-credentials.md

```diff
@@ -1,9 +1,324 @@
+regular expression matching the repository URL.
+
+When searching for an exact match, and then again when searching for a pattern
+match, appropriately labeled `Secret`s are considered in lexical order by name.
+:::
+
+:::info
+Because Kargo matches credentials to repositories by repository type and URL,
+users do not need to be informed of the details (e.g. names) of any global
+credentials, except possibly that they exist.
+:::
+
+### Enabling Global Credentials
+
+To designate one or more namespaces as containing "global" credentials, list
+them under the Kargo Helm chart's `controller.globalCredentials.namespaces`
+option at installation time.
+
+Operators must also manually ensure Kargo controllers receive read-only access
+to `Secret`s in the designated namespaces. For example, if `kargo-global-creds`
+is designated as a global credentials namespace, the following `RoleBinding`
+should be created within that namespace:
+
+```yaml
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+    name: kargo-controller-read-secrets
+    namespace: kargo-global-creds
+subjects:
+    - kind: ServiceAccount
+      name: kargo-controller
+      namespace: kargo
+roleRef:
+    kind: ClusterRole
+    name: kargo-controller-read-secrets
+    apiGroup: rbac.authorization.k8s.io
+```
+
+:::note
+The `kargo-controller-read-secrets` `ClusterRole` is predefined by the Kargo
+Helm chart and grants read-only access to `Secret` resources.
+:::
+
+:::info
+By default, Kargo controllers _lack_ cluster-wide permissions on `Secret`
+resources. Instead, the Kargo _management controller_ dynamically expands and
+contracts controller access to `Secret`s on a namespace-by-namespace basis as
+new `Project`s are created and deleted.
+
+_It is because this process does not account for "global" credential namespaces
+that these bindings must be created manually by an operator._
+:::
+
 :::warning
-Placeholder
+Setting the `controller.serviceAccount.clusterWideSecretReadingEnabled` option
+to `true` at installation will grant Kargo controllers cluster-wide read
+permission on `Secret` resources.
+
+__This is highly discouraged, especially in sharded environments where this
+permission would have the undesirable effect of granting remote Kargo
+controllers read permissions on all `Secret`s throughout the Kargo control
+plane's cluster -- including `Secret`s having nothing to do with Kargo.__
 :::
+
+## Other Forms of Credentials
+
+This section provides guidance on configuring Kargo and various cloud platforms
+to support "ambient" credentials. Kargo users are presumed not to have
+sufficient access to those platform to configure these options themselves, so
+this section is intended for operators and cloud platform administrators.
+
+### Amazon Elastic Container Registry (ECR)
+
+This section covers configuring Kargo to authenticate to ECR repositories using
+EKS Pod Identity _or_ IAM Roles for Service Accounts (IRSA).
```

Comment:
I feel something like

> Kargo can be configured to authenticate [...]

brings the same point across while being more pleasant to read. The same applies to the sections below.

Reply:
Fixed in 13419ce850278baae5bda4a5fa61e8fc7bc8898e

lmk if it looks ok.